public/pages/Detectors/components/UpdateAlertConditions/UpdateAlertConditions.tsx

```diff
@@ -178,6 +201,7 @@ export default class UpdateAlertConditions extends Component<
           rulesOptions={rulesOptions}
           changeDetector={this.changeDetector}
           updateDataValidState={() => {}}
+          hasNotificationPlugin={this.state.plugins.includes(OS_NOTIFICATION_PLUGIN)}
```

Comment:
In my opinion, it might make more sense to pass the list of installed plugins (i.e., a `string[]`) through the props than a `hasNotificationPlugin` boolean. The reason being that, like security analytics, notifications has a back and frontend plugin. Users are not required to have both installed, so it's possible for just the backend to be installed without the frontend and vise versa. In such a situation, we would want the [channel selection combo box](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/136/files#diff-fb4d9e38297485bb3a31b9a63e72a9833848dc7b160b65640a7abe85077cf7f6R415) to function, but the [manage channels button](https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/136/files#diff-fb4d9e38297485bb3a31b9a63e72a9833848dc7b160b65640a7abe85077cf7f6R435) would redirect to a non-existent page.

Reply:
@AWSHurneyt yap we found that edge case of missing frontend plugin, but as I understood in discussion with @amsiglan we ignore it since that would be already "power user" whose knows what he is doing.

If this is not the case, let me know how we can check if frontend plugin is installed (I've researched on it, but haven't found the way) and then we could have all edge cases covered. 

Just for cleaner picture the moment we have 4 cases working in this way:
1. Alerting plugin: Backend 🔴 , Frontend 🔴 -> Red notification **shown** , combobox **disabled**, button **disabled**
3. Alerting plugin: Backend 🔴 ,Frontend 🟢 -> Red notification **shown** , combobox **disabled**, button **disabled**
4. Alerting plugin: Backend 🟢  ,Frontend 🟢 -> Red notification **not shown**, combobox **enabled**, button **enabled**
5. Alerting plugin: Backend 🟢  ,Frontend 🔴  -> Red notification **not shown**, combobox **enabled**, button **enabled + redirect to 404**

P.S. I will change to passing installedPlugins instead of hasNotificationPlugin, but do not understand how to fix missing FE plugin.

Reply:
@AWSHurneyt Do you know of a way to check whether the frontend plugin is installed? If not, I can look into it. I wouldn't block this PR on that though, so we can keep the current behavior where we enable/disable the "combo-box/manage-btn" pair based on backend plugin

Reply:
Ah, I thought `_cat/plugins` returned the installed frontend plugins as well; but it doesn't seem so. Agreed, let's not block for this reason.

Reply:
I commented on the issue for this task explaining that we can take this up as a follow-up item.